autopilot/choice_test.go

```diff
@@ -32,40 +34,79 @@ func TestChooseNEmptyMap(t *testing.T) {
 	}
 }
 
+// candidateMapVarLen is a type we'll use to generate maps of various lengths
+// up to 255 to be used during QuickTests.
+type candidateMapVarLen map[NodeID]*AttachmentDirective
+
+// Generate generates a value of type candidateMapVarLen to be used during
+// QuickTests.
+func (candidateMapVarLen) Generate(rand *rand.Rand, size int) reflect.Value {
```

Comment:
Nice, yeah I think the `Generate` router with custom types is a bit easier than specifying the value generating function each time in the main config.